sourcemash/frontend/static/js/views/items_view.js

```diff
@@ -14,13 +15,21 @@ Sourcemash.Views.ItemsView = Backbone.View.extend({
     },
 
     markAllAsRead: function() {
+        this.stopListening(this.collection, 'change:unread');
+        this.model.save({unread: false});
```

Comment:
Do we need these two lines? If someone clicks "markAllAsRead" multiple times, it's still going to retrigger the api call because of the save.